package.json

```diff
@@ -4,11 +4,13 @@
   "repository": "https://github.com/dimagi/commcare-hq",
   "description": "CommcareHQ Javascript tests",
   "dependencies": {
+    "biginteger": "^1.0.3",
```

Comment:
This is also because of a quirk in js-xpath. It uses `require('biginteger')` which doesn't work when `biginteger` lives at `js-xpath/lib/biginteger`. Since I was considering modifying js-xpath out of bounds (thinking most of it was auto-generated), installing it at `biginteger` through npm was a workaround.